### PR TITLE
zdeploy init

### PR DESCRIPTION
**Pull request by hjacobs**
First version of `zdeploy init` with just one simple "hello world" webapp template.

Addresses #45.

### CODE REVIEW

**Comment by hjacobs** [comment]
:+1:

**Comment by hjacobs** [comment]
Example run:
```
$ python3 -m zalando_deploy_cli init /tmp/test
Writing /tmp/test/NOTES.txt.. OK
Writing /tmp/test/deployment.yaml.. OK
Writing /tmp/test/apply/ingress.yaml.. OK
Writing /tmp/test/apply/service.yaml.. OK
Writing /tmp/test/pipeline.yaml.. OK

Please change the following files according to your needs:

* pipeline.yaml: set target AWS account and cluster ID
* deployment.yaml: change Docker image and environment variables
* apply/ingress.yaml: modify the DNS name and SSL certificate ARN

File: zalando_deploy_cli/cli.py [hunk 1]
```diff
@@ -13,7 +13,9 @@
 import yaml
 import pierone.api
 import zign.api
-from clickclick import AliasedGroup, error, info, print_table
+from clickclick import Action, AliasedGroup, error, info, print_table
+
```
Comment (tuxlife) [inline zalando_deploy_cli/cli.py]: Please remove the empty line

**Comment by hjacobs** [comment]
:+1:

**Comment by tuxlife** [comment]
:+1:

**Pull request merged**